parse_test.go

```diff
@@ -116,3 +116,47 @@ func TestParseFile(t *testing.T) {
 	}
 	assert.Equal(t, expected, cmds)
 }
+
+func TestParseReaderHeredocs(t *testing.T) {
+	dockerfile := `RUN <<EOF
+source $HOME/.bashrc && echo $HOME
+echo "Hello" >> /hello
+echo "World!" >> /hello
+EOF
+`
+	cmds, err := ParseReader(bytes.NewBufferString(dockerfile))
+	assert.Nil(t, err)
+	expected := []Command{
+		Command{
+			Cmd:       "RUN",
+			Original:  dockerfile,
+			StartLine: 1,
+			EndLine:   5,
+			Flags:     []string{},
+			Value:     []string{"<<EOF", "source $HOME/.bashrc && echo $HOME\necho \"Hello\" >> /hello\necho \"World!\" >> /hello\nEOF\n"},
```

Comment:
this doesn't seem correct.  the `<<EOF` is just syntax and shouldn't be part of the actual command right?